docs/content/direct/observability.md

```diff
@@ -30,12 +31,12 @@ Some KubeStellar components expose Go's built-in pprof debug endpoints for profi
 | kubestellar-controller-manager   | HTTP     | 8082  | /debug/pprof/   | None        |  |
 | ks-transport-controller         | HTTP     | 8092  | /debug/pprof/   | None        |  |
 | status-addon-controller         | HTTP     | 9282  | /debug/pprof/   | None        |  |
-| status-agent-controller         | HTTP     | 8082  | /debug/pprof/   | None        |  |
+| status-agent-controller         | HTTP     | 8083  | /debug/pprof/   | None        |  |
```

Comment:
The term is "Status Add-On Agent", not "Status Agent Controller". See https://rishi-jat.github.io/kubestellar/doc-observability-fix/direct/architecture/#kubestellar-architecture .